docs/getting_started/hopsworksai/guides/getting_started.rst

```diff
@@ -206,12 +206,13 @@ subnet and security group. Select *Deploy*.
 .. note::
     We recommend that you always configure an SSH key under advanced options to ensure you can troubleshoot the instance if necessary.
 
-.. _configure-instance.png: ../../../_images/configure-instance.png
-.. figure:: ../../../imgs/hopsworksai/configure-instance.png
-    :alt: Configuring the Hopsworks cluster
-    :target: `configure-instance.png`_
-    :align: center
-    :figclass: align-center
+..
```

Comment:
Is this extra? (..)

Reply:
That image doesn't exist, so i added the comment character (in green above). I removed the whole  image now.